creation/web_base/condor_startup.sh

```diff
@@ -880,6 +880,9 @@ EOF
                     cat >> "$CONDOR_CONFIG" <<EOF
 # Declare resource: ${i}
 MACHINE_RESOURCE_${res_name} = ${res_num}
+# Declare no use of GPUs, i.e. the worker node (WN) should not have GPUs.
+# Even though the WN may physically have them, we want them to be ignored.
+MACHINE_RESOURCE_GPUs = 0
```

Comment:
This may run in the loop and be printed multiple times, even if GPUs are present

Reply:
This ended up being the case and has been modified as needed.